frontend/App.tsx

```diff
@@ -1,19 +1,36 @@
+import 'react-native-gesture-handler';
 import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
+import { StyleSheet, Text, View, SafeAreaView, StatusBar, Platform } from 'react-native';
+import { DefaultTheme, Provider as PaperProvider } from 'react-native-paper';
+import {HeaderNav} from './components/HeaderNav';
+import {headerStates, bottomStates} from './components/States';
+import { BottomNav } from './components/BottomNav';
+
+const theme = {
+  ...DefaultTheme,
+  roundness: 2,
+  colors: {
+    ...DefaultTheme.colors,
+    primary: '#fff',
+    accent: '#00E676',
+  },
+};
 
 export default function App() {
   return (
-    <View style={styles.container}>
-      <Text>Open up App.tsx to start working on your app!</Text>
-    </View>
+    <PaperProvider theme={theme}>
+        <SafeAreaView style={styles.container}>
+          <HeaderNav currState={headerStates.search}/>
+          <BottomNav currState={bottomStates.itemSelection}/>
+        </SafeAreaView>
+    </PaperProvider>
   );
 }
 
 const styles = StyleSheet.create({
   container: {
     flex: 1,
-    backgroundColor: '#fff',
-    alignItems: 'center',
-    justifyContent: 'center',
+    backgroundColor: '#FAFAFA',
+    // paddingTop: Platform.OS === "android" ? StatusBar.currentHeight : 0,
```

Comment:
I have had to do this on a lot of my React Native projects to get the positioning of the header right. If we don't actually need it, though, could you just erase the comment so that we don't have messy code in `master`?